functions/src/modules/SQR.functions.ts

```diff
@@ -303,118 +299,107 @@ export const processSubmission = functions.database
 /////////////////////////////////////////////////
 export const importSpreadSheetData = functions.https.onRequest(
   async (req, res) => {
-    const auth = await google.auth.getClient({
-      scopes: ['https://www.googleapis.com/auth/spreadsheets'],
-    });
-
-    const spreadsheetId = functions.config().sqr.spreadsheetId;
-    const spreadsheet = new GoogleSpreadsheet(spreadsheetId);
-
-    const token = await auth.getAccessToken();
-    spreadsheet.setAuthToken(token);
-
-    const getInfo = promisify(spreadsheet.getInfo);
-    const data = await getInfo();
-
-    let AllotmentsSheet, SubmissionsSheet;
-
-    data.worksheets.forEach(worksheet => {
-      if (worksheet.title === 'Submissions') SubmissionsSheet = worksheet;
-      if (worksheet.title === 'Allotments') AllotmentsSheet = worksheet;
-    });
-
-    const getSubmissions = promisify(SubmissionsSheet.getRows);
-
-    // submissions = Submissions sheet rows
-    const submissions = await getSubmissions();
-    submissions.forEach(row => {
-      const serial = row['submissionserial'];
+    const spreadsheetId = functions.config().sqr.spreadsheet_id;
+
+    // Regex for validating file names in both sheets
+    const audioFileRegex = /(\w+)-\d+.*/;
+
+    ////////////////////////
+    //     Submissions
+    ////////////////////////
+    const submissionsSheet = new GoogleSheets(spreadsheetId, ISoundQualityReportSheet.Submissions);
+    const submissionsRows = await submissionsSheet.__getRows();
```

Comment:
I'm confused by the method name. Looks like it is not intended to be called from outside. @adrienshen can you comment here?

Reply:
I was intending on removing the original `getRows` and set the name of this as `getRows` .. `__getRows` is just a temporary name